test/e2e/textInput.cy.js

```diff
@@ -0,0 +1,26 @@
+describe('Text Input', function () {
+  beforeEach(function () {
+    cy.getData()
+    cy.visit('/');
+  });
+
+  it('should display the text input component', function () {
+    const textInputComponents = this.data.components.filter((component) => component._component === 'textinput')
+    textInputComponents.forEach((textInputComponent) => {
```

Comment:
```suggestion
    textInputComponents.forEach(textInputComponent => {
```

These brackets aren't needed, please remove them from fat-arrow functions.

Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#syntax